.github/workflows/update-browserslist.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Browserslist DB
+on:
+  - workflow_call
+
+jobs:
+  update-dep:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
+        npm: [8.5.x]
+
+    steps:
+    - name: Check out the repo
+      uses: actions/checkout@v2
+      
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node }}
+    
+    - name: Install dependencies
+      run: npm install -g npm@${{ matrix.npm }}
+
+    - name: Install Packages 
+      run: npm ci
+
+    - name: Update dependencies
+      run: npx browserslist@latest --update-db
```

Comment:
Note: I just stumbled on https://github.com/c2corg/browserslist-update-action, which is a Github Action we might be able to use here instead. Worth noting that the README for that action mentions:

> v1 uses command `npx browserslist@latest --update-db` which is **now deprecated and to be removed in a future major browserslist release**.

It is being replaced with a standalone NPM package `update-browserslist-db` instead.